### ISSUE TITLE
Error when trying to Update Metadata for an Artist

### ISSUE THREAD

**Issue by vingerha** [music-assistant/hass-music-assistant#3104]
### What version of Music Assistant has the issue?

240-beta3 (also in beta2)

### What version of the Home Assistant Integration have you got installed?

n.a.

### Have you tried everything in the Troubleshooting FAQ and reviewed the Open and Closed Issues and Discussions to resolve this yourself?

- [X] Yes

### The problem

When selecting an artist, go to the 3-dots and select "Update metadata"
![image](https://github.com/user-attachments/assets/dfef7ab5-2f0a-4e9d-9f06-169db0a3ebd3)

### How to reproduce

see above

### Music Providers

local files

### Player Providers

DLNA / n.a.

### Full log output

```
...
2024-10-29 06:54:44.120 INFO (MainThread) [music_assistant.music] Sync task for Classical completed
2024-10-29 06:54:56.514 ERROR (MainThread) [music_assistant.webserver] Error handling message: metadata/update_metadata: 'library://artist/1161' is not in deque
2024-10-29 06:55:11.490 ERROR (MainThread) [music_assistant.webserver] Error handling message: metadata/update_metadata: 'library://artist/1161' is not in deque
2024-10-29 06:59:41.808 DEBUG (MainThread) [music_assistant.dlna] DLNA discovery started...
```

### What version of Home Assistant Core are your running

n.a.

### What type of installation are you running?

Home Assistant Container

### On what type of hardware are you running?

Generic x86-64 (e.g. Intel NUC)

**Comment by vingerha** [comment]
Adding: @OzGav reports that it happens on Albums as well

**Comment by OzGav** [comment]
Also just saw it when trying to re-add a manual radio station

**Comment by OzGav** [comment]
This has been fixed in 2.4.0b4

**Comment by vingerha** [comment]
Confirmed for Artist and Album, cannot verify other cases
